setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools, os
 
 PACKAGE_NAME = 'xt-training'
-VERSION = '1.13.0'
+VERSION = '1.14.0'
```

Comment:
If we are following SemVer, this should increment the major version since it is a breaking change (due to train_exit). But as I said in my other comment, I think we should change it to be backwards compatible, in which case this should stay as a minor version increment.